hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -1143,37 +1144,37 @@ private Log getLogForFungibleTransfer(final Address logger) {
 					.forDataItem(amount).build();
 		}
 
-		private Log getLogForNftExchange(final Address logger) {
-			final var nftExchanges = super.transferOp.get(0).nftExchanges();
-			final var nftExchange = nftExchanges.get(0).asGrpc();
-			final var sender = asTypedEvmAddress(nftExchange.getSenderAccountID());
-			final var receiver = asTypedEvmAddress(nftExchange.getReceiverAccountID());
-			final var serialNumber = nftExchange.getSerialNumber();
-
-			return EncodingFacade.LogBuilder.logBuilder().forLogger(logger)
-					.forEventSignature(TRANSFER_EVENT)
-					.forIndexedArgument(sender)
-					.forIndexedArgument(receiver)
-					.forIndexedArgument(serialNumber).build();
-		}
+//		private Log getLogForNftExchange(final Address logger) {
```

Comment:
I don't think long stretches of commented out code are good for production code.  We can come back to this PR if we need the code.